File: source/back-end/src/main/java/be/pxl/backend/service/IEmployeeService.java
package be.pxl.backend.service;

import java.util.List;
import be.pxl.backend.entity.Employee;

public interface IEmployeeService {

	public Employee find(int id);
	
	public List<Employee> all();
	
	public void persist(Employee employee);
	
	public void delete(int id);
	
	public void update(Employee employee);

	public Employee getEmployeeByUsername(String username);
}

File: Source/frontend/frontend.Service/IEmployeeService.cs
using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface IEmployeeService
    {
        List<Employee> All();
        Employee Find(int id);
        void Add(Employee employee);
        void Update(Employee employee);
        void Delete(int id);
    }
}
